styles/styles.css

```diff
@@ -1201,6 +1201,56 @@ main.blue-contract .section.section-with-title p {
   width: 1em;
 }
 
+/* Headings classes styles */
+.redesign-v2 .h2 {
+  font-family: var(--ff-volvo-novum);
+
+  /* font-size: 56px; */
+  font-size: var(--f-heading-2-font-size);
+  font-style: normal;
+  font-weight: 300;
+  line-height: 110%;
+  letter-spacing: -0.5px;
+  font-feature-settings: 'clig' off, 'liga' off;
+}
+
+.redesign-v2 h4 {
```

Comment:
Why the rest are `.redesign-v2 .h5` and this one is changing the tag directly?

Reply:
It is a typo :)
Fixed